apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -22,12 +22,16 @@ import {
   FieldKey,
   GeneratedEffect,
   MinMax,
-} from '../types';
+} from '@cdo/apps/dance/ai/types';
 import {
   generateBlocks,
   generateBlocksFromResult,
   generatePreviewCode,
+  getImageUrl,
```

Comment:
I think it could be good to rename this `getEmojiImageUrl` (something I meant to do a while ago) while we're moving it.